python/paddle/fluid/dygraph/jit.py

```diff
@@ -64,14 +65,56 @@ def __impl__(*args, **kwargs):
         # Transform AST
         dygraph_to_static = DygraphToStaticAst()
         root_wrapper = dygraph_to_static.get_static_ast(root)
+
+        # Get static_func from AST
         func_name = dygraph_to_static.get_module_name()
         static_func, file_name = ast_to_func(root_wrapper.node, func_name)
 
-        return static_func(*args, **kwargs)
+        if not in_dygraph_mode():
+            return static_func(*args, **kwargs)
+        else:
+            feed_name_to_idx = dygraph_to_static.get_feed_name_to_idx()
+            feed_dict = {}
+            for feed_name, idx in feed_name_to_idx.items():
+                feed_dict[feed_name] = args[idx]
+
+            # Run static_func in static mode
+            import paddle.fluid as fluid
+            startup_program = fluid.default_startup_program()
+            main_program = fluid.default_main_program()
+
+            static_res = run_static_func(main_program, startup_program,
+                                         static_func, args, kwargs, feed_dict,
+                                         feed_name_to_idx)
+
+        return static_res
 
     return __impl__
 
 
+@switch_to_static_graph
+def run_static_func(main_program, startup_program, static_func, args, kwargs,
+                    feed_dict, feed_name_to_idx):
+    import paddle.fluid as fluid
```

Comment:
Same comment, could we delete "import paddle.fluid as fluid" here and put it at the header?

Reply:
Yes. Done